tiller_python_service/git_actions.py

```diff
@@ -202,6 +202,9 @@ def merge_pull_request(pull_request_url, commit_title, commit_message, merge_met
     else:
         print(f'Failed to merge pull request. Response: {response.content}')
```

Comment:
We shouldn't be using raw prints in this file, can you update it to use the `logging` library? help tiller

Reply:
We shouldn't be using raw prints in this file, can you update it to use the `logging` library? help tiller

Reply:
We shouldn't be using raw prints in this file, can you update it to use the `logging` library? help tiller